docs/src/usage/std.md

```diff
@@ -67,15 +67,22 @@ This field is used for allowing smarter introspection of how the argument given
 - "read" - This argument is only read from. Currently unused.
 - "write" - This argument is only written to. Used by `unused_variable` to assist in detecting a variable only being written to, even if passed into a function.
 
+#### "must_use"
+This field is used for checking if the return value of a function is used.
+
+- `true` - The default. The return value of this function must be used.
```

Comment:
The default is false.

Reply:
Resolved in: https://github.com/Kampfkarren/selene/pull/583/commits/b7c23500aa029dff2ea99af1d2c18af418bcd4de